core/vision/zed_handler.py

```diff
@@ -17,21 +19,25 @@ def __init__(self):
         self.feed_handler = feed_handler
         self.logger = logging.getLogger(__name__)
 
+        # Define the camera resolutions
+        self.depth_res_x = 640
+        self.depth_res_y = 360
+        self.reg_res_x = 1280
+        self.reg_res_y = 720
+        self.hfov = 85
+
+        # Define the desired runtime FPS
+        self.fps = 30
```

Comment:
Do we want to keep 30fps for autonomy quality reasons, or are we considering dropping down to 15 to match the spinels

Reply:
This is a good question, I was planning on examining our framerates in a testing situation and see. I'll make an issue to track it.